Exercise-HTML-CSS-1/index.html

```diff
@@ -0,0 +1,145 @@
+            </div>
+                
+            </div>
+        </div> </br>
+
+		<!--div location-->
+		<div>
+			<h2>Our Locations</h2>
+				<ul>
+					<li>USA
+						<ul>
+								<li>
+									<a href ="https://www.girldevelopit.com/chapters/new-york">New York</a>
+								</li>
+
+								<li>
+									<a href ="https://www.girldevelopit.com/chapters/columbus">Columbus</a>
+								</li>
+
+								<li>
+									<a href ="https://www.girldevelopit.com/chapters/austin">Austin</a>
+								</li>
+
+								<li>
+									<a href ="https://www.girldevelopit.com/chapters/philadelphia">Philly</a>
+								</li>
+							</ul>
+						</li>
+
+						<li>World
+							<ul>
+								<li>
+									<a href = "https://www.girldevelopit.com/chapters/ottawa">Ottawa</a>
+								</li>
+
+								<li>
+									<a href = "https://www.meetup.com/es-ES/topics/girl-develop-it/au/sydney/?country=au&zipstatecity=sydney&state=&radius=25">Sydney</a>
+								</li>
+							</ul>
+						</li>
+					</ul>
+				</div>
+
+				<!--Classes div-->
+				<div>
+					<h2>Upcoming Classes</h2>
+
+					<table>
+						<tr>
+							<th>Location</th>
+							<th>Topic</th>
+						</tr>
+
+						<tr>
+							<td>New York</td>
+							<td>JavaScript & jQuery</td>
+						</tr>
+
+						<tr>
+							<td>San Francisco</td>
+							<td>HTML & CSS</td>
+						</tr>
+					</table>
+                </div> </br>
+        
+                <!--Sign Up div-->
+                <div>
+                    <h2>Sign Up!</h2>
+                    
+                    <label>
+                        Name:
+                        <input type = "text" name="Name">
+                    </label> </br>
+        
+                    <label>
+                        Email:
+                        <input type = "text" name = "Email">
+                    </label> </br>
+        
+                    <label for = "Location">Location:</label> 
+                    <select name = "Location" id = "Location">
+                        <option value = "SF">San Francisco</option>
+                        <option value = "NY">New York</option>
+                    </select>
+
+                    <fieldset class = "Gender-fs">
+                        <legend>Gender:</legend>
+                        <label for = "Female">Female</label>
+                        <input type = "radio" name = "gender" id = "Female" value = "Female">
+                        <label for = "Male">Male</label>
+                        <input type = "radio" name = "gender" id ="Male" value = "Male">
+                    </fieldset>
+
+                    <label>
+                        Experience: </br>
+                        <textarea name ="experience"></textarea>
+                    </label> </br>
+
+                    <button type ="submit">Sign Up</button>
+                    
+                </div> </br>
+
+				<!--Social div-->
+				<div>
```

Comment:
Maybe use a `<footer>` tag here. It won't make any difference in the rendering, but it makes your code more understandable.